neutron-physics/model.py

```diff
@@ -0,0 +1,387 @@
+
+        """
+        if re.match('7[0-4]c', self.suffix):
+            # Get the table from the ENDF/B-VII.0 neutron ACE files
+            name = zaid(self.nuclide, self.suffix)
+            for path in self.library.glob('endf70[a-k]'):
+                try:
+                    table = openmc.data.ace.get_table(path, name)
+                except ValueError:
+                    pass
+        else:
+            # Get the table from the ENDF/B-VII.1 neutron ACE files
+            Z, A, m = openmc.data.zam(self.nuclide)
+            element = openmc.data.ATOMIC_SYMBOL[Z]
+            name = szax(self.nuclide, self.suffix)
+            path = self.library / 'endf71x' / f'{element}' / f'{name}'
+            table = openmc.data.ace.get_table(path, name)
+
+        # Convert cross section data
+        data = openmc.data.IncidentNeutron.from_ace(table, 'mcnp')
+        self._temperature = data.kTs[0] / openmc.data.K_BOLTZMANN
+
+        # Export HDF5 file
+        os.makedirs('openmc', exist_ok=True)
+        h5_file = Path('openmc') / f'{data.name}.h5'
+        data.export_to_hdf5(h5_file, 'w')
+
+        # Register with library and write cross_sections.xml
+        data_lib = openmc.data.DataLibrary()
+        data_lib.register_file(h5_file)
+        data_lib.export_to_xml(Path('openmc') / 'cross_sections.xml')
+
+    def _make_openmc_input(self):
+        """Generate the OpenMC input XML
+
+        """
+        # Directory from which openmc is run
+        os.makedirs('openmc', exist_ok=True)
+        
+        # Define material
+        mat = openmc.Material()
+        mat.add_nuclide(self.nuclide, 1.0)
+        mat.set_density('g/cm3', self.density)
+        materials = openmc.Materials([mat])
+        if self.library is not None:
+            xs_path = (Path('openmc') / 'cross_sections.xml').resolve()
+            materials.cross_sections = str(xs_path)
+        materials.export_to_xml(Path('openmc') / 'materials.xml')
+
+        # Set up geometry
+        sphere = openmc.Sphere(boundary_type='reflective', R=1.e9)
+        cell = openmc.Cell(fill=materials, region=-sphere)
+        geometry = openmc.Geometry([cell])
+        geometry.export_to_xml(Path('openmc') / 'geometry.xml')
+
+        # Define source
+        source = openmc.Source()
+        source.space = openmc.stats.Point((0,0,0))
+        source.angle = openmc.stats.Isotropic()
+        source.energy = openmc.stats.Discrete([self.energy], [1.])
+
+        # Settings
+        settings = openmc.Settings()
+        if self._temperature is not None:
+            settings.temperature = {'default': self._temperature}
+        settings.source = source
+        settings.particles = self.particles
+        settings.run_mode = 'fixed source'
+        settings.batches = 1
+        settings.create_fission_neutrons = False
+        settings.export_to_xml(Path('openmc') / 'settings.xml')
+ 
+        # Define tallies
+        energy_bins = np.logspace(3, np.log10(self.energy), 500)
```

Comment:
For neutrons, we need to go much lower than 1 keV. I would probably tally all the way down to 1e-5 eV as that is generally the lowest point in the data.